core/src/main/java/org/elasticsearch/plugins/RemovePluginCommand.java

```diff
@@ -56,7 +56,18 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
         execute(terminal, arg, env);
     }
 
-    // pkg private for testing
+    /**
+    * Remove the plugin named {@code plugin named}.
+    *
+    * @param terminal       the Terminal to use for input/output
+    * @param pluginName     the name of the plugin to remove.
+    * @param env            the environment for the local node, which may be used for the local settings and path
+    *
+    * @throws UserException if plugin name is null.
```

Comment:
We can remove the period here, this is a sentence fragment (as these usually are).